docs/data_access.rst

```diff
@@ -115,3 +115,24 @@ on the collection and the catalog that is used by it. This is not a problem caus
 standardization between catalogs of EO data.
 
 
+Handling large vector data
+#######################
+
+Handling large volumes of data is crucial for making decisions, improving processes and having an efficient solution. 
+However, if you attempt to execute a process on significantly large data, you would probably encounter a 'Request Entity Too Large' 
```

Comment:
> attempt to execute a process on significantly large data, you would probably encounter a 'Request Entity Too Large'

I think this is a bit misleading: openEO, by design, allows to work on large (raster) data sets. The problem is that your REST requests should not be too large (e.g. avoid embedding large vector data sets directly in the process graph)